content/en/docs/release_notes/york.md

```diff
@@ -17,6 +17,7 @@ Brief summary of the notable release contents.
 ## Fixes
 
 * #7100: Fixed an issue installing [@axway/amplify-api-builder-cli](https://www.npmjs.com/package/@axway/amplify-api-builder-cli) version `1.20.0` that resulted in the error `error syscall spawn git` when installing "git://github.com/wickedest/inquirer-checkbox-plus-prompt.git".
+* #7081: Previously, the Model API responses were logging body at INFO level. Now, body is not been logged in the response as part of improving performance.
```

Comment:
**grammar**: "body is not been logged".  You're mixing tenses.  "is" is present tense, "been" is past-tense, but also not needed.  Also note that it's not just model API ,but custom API too.  Your unit-test fix was to the testapi.
```suggestion
* #7081: Previously, model and custom API logged the response body at INFO level. Now, model and custom API do not log the response body at any level.
```